app/js/lib/foxbox/api.js

```diff
@@ -9,16 +9,16 @@ const p = Object.freeze({
 
   watchTimer: Symbol('watchTimer'),
   watchEventBus: Symbol('watchEventBus'),
-  watchGetters: Symbol('watchGetters'),
+  watchChannels: Symbol('watchChannels'),
 
   // Private methods.
   getURL: Symbol('getURL'),
   onceOnline: Symbol('onceOnline'),
   onceAuthenticated: Symbol('onceAuthenticated'),
   onceDocumentVisible: Symbol('onceDocumentVisible'),
   onceReady: Symbol('onceReady'),
-  fetchGetterValues: Symbol('fetchGetterValues'),
-  updateGetterValue: Symbol('updateGetterValue'),
+  fetchChannelValues: Symbol('fetchChannelValues'),
+  updateChannelValue: Symbol('updateChannelValue'),
```

Comment:
I remember we used `get` and `set` in the service classes rather than `fetch` and `update`. Do you think we can use a consistent pattern all over the lib?